package.json

```diff
@@ -37,6 +37,7 @@
     "babel-plugin-transform-es3-property-literals": "6.8.0",
     "babel-plugin-transform-flow-strip-types": "^6.18.0",
     "babel-plugin-transform-inline-environment-variables": "6.8.0",
+    "babel-plugin-transform-runtime": "^6.23.0",
```

Comment:
we use this package just for tests here https://github.com/cssinjs/jss/pull/512/files#diff-004e32ee0ee65a48e2e1924046cec030